### PR TITLE
Going for a deploy...

### PR DESCRIPTION
**Pull request by zephraph**
This is a huge update. Has MP v2 stuff in it.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -27,7 +26,7 @@ import {
 import { middleware as requestIDsAdder } from "./lib/requestIDs"
 import { nameOldEigenQueries } from "./lib/modifyOldEigenQueries"
 import { rateLimiter } from "./lib/rateLimiter"
-import { graphqlErrorHandler } from "./lib/graphqlErrorHandler"
+// import { graphqlErrorHandler } from "./lib/graphqlErrorHandler"
```
Comment (alloy) [inline src/index.ts]: @zephraph Hmm, are we going to miss runtime exceptions being recorded in Sentry?

File: src/index.ts [hunk 2]
```diff
@@ -160,11 +154,12 @@ async function startApp() {
...
     })
 
-    server.applyMiddleware({ app, path: "/" })
+    server.applyMiddleware({ app, path })
   } else {
     const graphqlHTTP = require("express-graphql")
     app.use(
-      graphqlHTTP((req, res, params) => {
+      graphqlHTTP((req, res /*, params */) => {
+        console.log("Request from", path)
```
Comment (alloy) [inline src/index.ts]: Stray loggy?

Comment (zephraph) [reply src/index.ts]: Somewhat, yeah. It does tell if the request is coming from v1 or v2 which is useful for the logs.

**Comment by alloy** [comment]
Based on skimming, this looks good to me. I’m only afraid that we’re losing sentry reporting, which may be something we want especially with this big release?

**Comment by zephraph** [comment]
Why would we be losing sentry reporting?

**Comment by alloy** [comment]
Because that’s done from the error handler https://github.com/artsy/metaphysics/blob/4447c0f2d2d25f67f62e7a7a5758370d460bc019/src/lib/graphqlErrorHandler.ts#L71-L81

**Pull request merged**